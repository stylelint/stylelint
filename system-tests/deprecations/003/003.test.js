/* @flow */
"use strict"

const systemTestUtils = require("../../systemTestUtils")
const stylelint = require("../../../lib")

it("deprecations/003", () => {
  return stylelint.lint({
    files: [systemTestUtils.caseStylesheetGlob("deprecations/003")],
    configFile: systemTestUtils.caseConfig("deprecations/003"),
  }).then((output) => {
    expect(systemTestUtils.prepResults(output.results)).toMatchSnapshot()
  })
})
