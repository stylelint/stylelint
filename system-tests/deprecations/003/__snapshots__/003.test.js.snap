// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deprecations/003 1`] = `
Array [
  Object {
    "deprecations": Array [
      Object {
        "reference": "https://stylelint.io/user-guide/rules/function-url-data-uris/",
        "text": "'function-url-data-uris' has been deprecated and in 8.0 will be removed. Instead use either the 'function-url-scheme-blacklist' or 'function-url-scheme-whitelist' rule.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-no-attribute/",
        "text": "'selector-no-attribute' has been deprecated and in 8.0 will be removed. Instead use 'selector-max-attribute' with '0' as its primary option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-no-combinator/",
        "text": "'selector-no-combinator' has been deprecated and in 8.0 will be removed. Instead use 'selector-max-combinators' with '0' as its primary option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-no-id/",
        "text": "'selector-no-id' has been deprecated and in 8.0 will be removed. Instead use 'selector-max-id' with '0' as its primary option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-no-type/",
        "text": "'selector-no-type' has been deprecated and in 8.0 will be removed. Instead use 'selector-max-type' with '0' as its primary option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-no-universal/",
        "text": "'selector-no-universal' has been deprecated and in 8.0 will be removed. Instead use 'selector-max-universal' with '0' as its primary option.",
      },
    ],
    "errored": undefined,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "source": "deprecations/003/stylesheet.css",
    "warnings": Array [],
  },
]
`;
